trie/statistics/trieStatistics.go

```diff
@@ -152,9 +120,8 @@ func (tsd *TrieStatsDTO) ToString() []string {
 	stats = append(stats, fmt.Sprintf("branch nodes size %v,", core.ConvertBytes(tsd.BranchNodesSize)))
 	stats = append(stats, fmt.Sprintf("extension nodes size %v,", core.ConvertBytes(tsd.ExtensionNodesSize)))
 	stats = append(stats, fmt.Sprintf("leaf nodes size %v,", core.ConvertBytes(tsd.LeafNodesSize)))
-	stats = append(stats, fmt.Sprintf("total nodes per level %v,", tsd.TotalNumNodesPerLevel))
-	stats = append(stats, fmt.Sprintf("num branches per level %v,", tsd.NumBranchesPerLevel))
-	stats = append(stats, fmt.Sprintf("num extensions per level %v,", tsd.NumExtensionsPerLevel))
-	stats = append(stats, fmt.Sprintf("num leaves per level %v", tsd.NumLeavesPerLevel))
+	stats = append(stats, fmt.Sprintf("num branches  %v,", tsd.NumBranchNodes))
```

Comment:
two spaces in the next 3 lines before %v, intended?

Reply:
No, copy-paste error. Will fix.